### PR TITLE
feat(FN-974): create environment variable pdc inputter email

### PR DESCRIPTION
**Pull request by francescastocco**
## Introduction

We need an environment variable for PDC Inputters email

## Resolution
- Added to Bicep
- Add to github actions

### CODE REVIEW

File: .github/workflows/bicep_workflow_feature.yml [hunk 1]
```diff
@@ -4,6 +4,7 @@ on:
   push:
     branches:
       - feat/stbr-infra
+      - feat/FN-974/create-environment-variable-pdc-inputter-email
     paths:
       - '.github/workflows/bicep_workflow.yml'
       - '.github/workflows/bicep_deploy.yml'
```
Comment (francescastocco) [inline .github/workflows/bicep_workflow_feature.yml:10]: should these be this instead?
```suggestion
      - '.github/workflows/bicep_workflow_feature.yml'
      - '.github/workflows/bicep_deploy.yml'
      - '.github/workflows/bicep_az_deployment_group.yml'
```

File: .github/workflows/bicep_workflow_feature.yml [hunk 2]
```diff
@@ -4,6 +4,7 @@ on:
   push:
     branches:
       - feat/stbr-infra
+      - feat/FN-974/create-environment-variable-pdc-inputter-email
```
Comment (ChristianMcc) [inline .github/workflows/bicep_workflow_feature.yml:7]: Can we remove this merging?

**Review by ChristianMcc — CHANGES_REQUESTED** [review]

**Review by OxonianCambion — APPROVED** [review]

File: .github/workflows/bicep_az_deployment_group.yml [hunk 3]
```diff
@@ -228,4 +231,6 @@ jobs:
               AZURE_NUMBER_GENERATOR_FUNCTION_SCHEDULE="$AZURE_NUMBER_GENERATOR_FUNCTION_SCHEDULE_ENV" \
               SESSION_SECRET=${{ secrets.SESSION_SECRET }} \
               ESTORE_URL=${{ secrets.ESTORE_URL }} \
-              RATE_LIMIT_THRESHOLD=${{ vars.RATE_LIMIT_THRESHOLD}}
+              PDC_INPUTTERS_EMAIL_RECIPIENT=${{ secrets.PDC_INPUTTERS_EMAIL_RECIPIENT }} \
+              RATE_LIMIT_THRESHOLD=${{ vars.RATE_LIMIT_THRESHOLD}} \
+              MAX_UTILISATION_REPORT_FILE_SIZE=${{ vars.MAX_UTILISATION_REPORT_FILE_SIZE }}
```
Comment (OxonianCambion) [inline .github/workflows/bicep_az_deployment_group.yml:236]: Make sure this variable exists in GH's `dev` environment.

File: .github/workflows/bicep_workflow_feature.yml [hunk 4]
```diff
@@ -75,5 +75,6 @@ jobs:
       AZURE_NUMBER_GENERATOR_FUNCTION_SCHEDULE: ${{ secrets.AZURE_NUMBER_GENERATOR_FUNCTION_SCHEDULE }}
       SESSION_SECRET: ${{ secrets.SESSION_SECRET }}
       ESTORE_URL: ${{ secrets.ESTORE_URL }}
+      PDC_INPUTTERS_EMAIL_RECIPIENT: ${{ secrets.PDC_INPUTTERS_EMAIL_RECIPIENT }}
```
Comment (OxonianCambion) [inline .github/workflows/bicep_workflow_feature.yml:78]: Make sure this secret exists in `dev`

**Review by ChristianMcc — APPROVED** [review]

**Pull request merged**